docs/quickstart.rst

```diff
@@ -7,15 +7,15 @@ Quick Start
 -  `Mail Configuration <#mail-configuration>`_
 
 Basic SQLAlchemy Application
-=============================
+============================
 
 SQLAlchemy Install requirements
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 ::
 
      $ mkvirtualenv <your-app-name>
-     $ pip install flask-security flask-sqlalchemy
+     $ pip install flask-security sqlalchemy
```

Comment:
We should keep the less chatty example with flask-sqlalchemy in the documentation and the "advance" one after.